### PR TITLE
Added centos based dockerfiles and cccp.yml.

### PR DESCRIPTION
**Pull request by mohammedzee1000**
This is for the purpose of building these containers through
the centos community container pipeline.

### CODE REVIEW

**Comment by petervo** [comment]
We are depreciating this repo and moving the docker file to https://github.com/cockpit-project/cockpit/tree/master/containers/ws. Could you open your PR against that docker file?

**Comment by petervo** [comment]
Turns out I was wrong and we are keeping this repo as the base.

File: Dockerfile.centos7 [hunk 1]
```diff
@@ -0,0 +1,29 @@
+FROM registry.centos.org/centos/centos
+MAINTAINER "Stef Walter" <stefw@redhat.com>
+
+RUN yum -y update
+
+ENV INSTALL_PKGS "cockpit cockpit-ws"
```
Comment (petervo) [inline Dockerfile.centos7]: only cockpit-ws should be installed.

File: Dockerfile.centos7 [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+# Get this specific version of cockpit-ws
+RUN yum -y install ${INSTALL_PKGS} && yum clean all
```
Comment (petervo) [inline Dockerfile.centos7]: would be nice to respect versions

File: Dockerfile.centos7 (hunk 1 above)
Comment (mohammedzee1000) [inline Dockerfile.centos7]: I will update the PR accordingly.

**Comment by petervo** [comment]
Replaced by https://github.com/cockpit-project/cockpit/pull/5539

If we end up needing it here, we can port those changes over.

**Pull request closed without merging**